modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx

```diff
@@ -159,13 +159,18 @@ const SaveFDSFieldsModalContent = ({
 	const saveFDSFields = async () => {
 		setSaveButtonDisabled(true);
 
-		const creationData: Array<{name: string; type: string}> = [];
+		const creationData: Array<{
+			name: string;
+			sortable: boolean;
+			type: string;
+		}> = [];
```

Comment:
The code in `SaveFDSFieldsModalContent` seems good, but we're still missing the same code in `AddFieldsModalContent`. It's 
```js
if (Liferay.FeatureFlags['LPS-186871']) {
    // AddFieldsModalContent
}
else {
    // SaveFDSFieldsModalContent
}
``` 
`Add*` implements treeview selection, `Save*` is the old list selection. The naming is a bit awkward, but :man_shrugging: We'll clean it up when we remove FF.

I believe the code as it is in this PR sets all fields as not sortable by default.